test/NuGet.Core.Tests/NuGet.CommandLine.XPlat.Test/BasicLoggingTests.cs

```diff
@@ -69,6 +69,29 @@ public void BasicLogging_RestoreHelp_ExitCode()
         }
 
         [Fact]
+        public void BasicLogging_RestoreConfigFile_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
```

Comment:
Logged https://github.com/NuGet/Home/issues/2040 for this issue.

Is this PR ok to merge?